src/test/ui/command/command-setgroups.rs

```diff
@@ -0,0 +1,16 @@
+// ignore-android
```

Comment:
By default ui tests are expected to fail compilation. To test that program runs succesfully it needs `run-pass` annotation:

```
// run-pass
```

Additionaly I would copy some of ignore directives found in other tests:

```
// ignore-windows - this is a unix-specific test
// ignore-cloudabi no processes
// ignore-emscripten no processes
// ignore-sgx no processes
```

Not sure about android, but test could also try to set more than NGROUPS_MAX to cause a failure regardless of premissions.